implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -150,6 +150,12 @@ impl<'a> Rpc<'a> {
                 // Overwrite error to swallow inner cause and hide it from end-user
                 miette!("The request timed out, please make sure the command's arguments are correct or try again")
             })?;
+
+        if self.is_ok().is_err() {
+            let err: Error = self.parse_response()?;
+            let err_msg = err.message().unwrap_or_default().to_string();
```

Comment:
I'm not entirely sure what a default error message would look like here.

It could be anything from an internal error, to a resource not found, or authorization issue.

The calling code, at this point could provide more context as to what failed, but not why.

Reply:
> The calling code, at this point could provide more context as to what failed, but not why.

Yeah, that makes sense.